.env.template

```diff
@@ -1,7 +0,0 @@
-# Flask server configuration.
-FLASK_APP=todo_app/app
-FLASK_ENV=development
-
-# Change the following values for local development.
-SECRET_KEY=secret-key
-PRIMARY_CONNECTION=mongodb://module10:ulH6uBzQRqsLbVIWVR44Cil6nOwb6AML7ykXzCCBygQI4uvkn2Ok8yK1B3GjrhzCgjOE3LdGHJhkUObXOtpXaw==@module10.mongo.cosmos.azure.com:10255/?ssl=true&replicaSet=globaldb&retrywrites=false&maxIdleTimeMS=120000&appName=@module10@
```

Comment:
We want to remove the values from this, but we don't want to just lose everything in the file and have it be empty! The idea is that this file gives a new dev a quick reminder of what variables need setting, specifies the format, and gives some sensible default values where possible
```suggestion
# Flask server configuration.
FLASK_APP=todo_app/app
FLASK_ENV=development

# Change the following values for local development.
SECRET_KEY=replace-with-a-secret-key
PRIMARY_CONNECTION=replace-with-your-connection-string
```

Reply:
done

Reply:
done